PhysicsTools/PatUtils/interface/SmearedJetProducerT.h

```diff
@@ -200,6 +202,13 @@ class SmearedJetProducerT : public edm::stream::EDProducer<> {
                     resolution = JME::JetResolution::get(setup, m_jets_algo_pt);
                     resolution_sf = JME::JetResolutionScaleFactor::get(setup, m_jets_algo);
                 }
+
+                if(m_useDeterministicSeed) {
+                    unsigned int runNum_uint = static_cast <unsigned int> (event.id().run());
+                    unsigned int evNum_uint = static_cast <unsigned int> (event.id().event()); 
+                    std::uint32_t seed = runNum_uint + 4*evNum_uint;
+                    m_random_generator.seed(seed);
```

Comment:
@slava77 this approach is copied from https://github.com/cms-sw/cmssw/blob/CMSSW_9_3_0_pre4/RecoJets/JetProducers/plugins/VirtualJetProducer.cc#L272, as noted in the event description. If it needs to be modified, I think it should be done coherently in a followup PR.